### ISSUE TITLE
Blocking: `directio.so: undefined symbol: aligned_allocate` in engine integration test

### ISSUE THREAD

**Issue by yasker** [longhorn/longhorn#968]
The latest Ubuntu 18.04 + Python 2.7 introduced a blocking issue for engine integration test:

```
data/test_restore_to_file.py:5: in <module>
--
14077 | from common import ( # NOQA
14078 | data/common.py:17: in <module>
14079 | from frontend import (
14080 | data/frontend.py:5: in <module>
14081 | import directio
14082 | E ImportError: /go/src/github.com/longhorn/longhorn-engine/integration/.tox/py27/local/lib/python2.7/site-packages/directio.so: undefined symbol: aligned_allocate
```

To reproduce:
```
$ docker run -it ubuntu:18.04 bash
root@f07c24add7d1:/# apt update
...
root@f07c24add7d1:/# apt install python-pip
...
root@f07c24add7d1:/# pip install directio
pCollecting directio
ython  Downloading https://files.pythonhosted.org/packages/6e/b0/5c4b0bcd375a9bf666cfe7db2ff8bb5d690c4c3bf1eb7be708fbdeb1dfe1/directio-1.2.tar.gz
 Building wheels for collected packages: directio
  Running setup.py bdist_wheel for directio ... done
  Stored in directory: /root/.cache/pip/wheels/f0/a8/85/706b44b4858cf01d5df7ae40580c2638ff06526b2b07159a1c
Successfully built directio
Installing collected packages: directio
Successfully installed directio-1.2

root@f07c24add7d1:/# python2
Python 2.7.17 (default, Nov  7 2019, 10:07:09)
[GCC 7.4.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import directio
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: /usr/local/lib/python2.7/dist-packages/directio.so: undefined symbol: aligned_allocate
```

The latest version still worked OK a couple of weeks back, and python 3 doesn't have this issue.

**Comment by yasker** [comment]
I've tried to pin python version back to `2.7.15-4ubuntu4~18.04.2` instead of `2.7.17`, but still doesn't work.

**Comment by yasker** [comment]
Workaround has been implemented in #439.

After #641 is done, we will revert the workaround and use 18.04 Python 3.

**Comment by yasker** [comment]
Done. We've reverted back to use 18.04 for Python 3.
